WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadStep.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.jetpack.backup
+
+import org.wordpress.android.util.wizard.WizardStep
+import javax.inject.Inject
+import javax.inject.Singleton
+
+enum class BackupDownloadStep : WizardStep {
+    DETAILS, PROGRESS, COMPLETE;
+
+    companion object {
+        fun fromString(input: String): BackupDownloadStep {
+            return when (input) {
+                "backup_download_details" -> DETAILS
+                "backup_download_progress" -> PROGRESS
+                "backup_download_complete" -> COMPLETE
+                else -> throw IllegalArgumentException("SiteCreationStep not recognized: \$input")
+            }
+        }
+    }
+}
+
+@Singleton
+class BackupDownloadStepsProvider @Inject constructor() {
+    fun getSteps(): List<BackupDownloadStep> {
```

Comment:
1) Minor (🔍 ): Consider simplifying the function to something like: `fun getSteps() = listOf(...)`
2) Question (❓ ): Does the list order matter?

Reply:
> Question (❓ ): Does the list order matter?

It appears that they do.

Reply:
> It appears that they do.

Thanks for clarifying that for me.

Reply:
in 6448ff8